src/EnergyPlus/WindowComplexManager.cc

```diff
@@ -2846,13 +2846,19 @@ namespace WindowComplexManager {
         nmix(nlayer + 1) = 1;      // pure air on indoor side
 
         // Simon: feed gas coefficients with air.  This is necessary for tarcog because it is used on indoor and outdoor sides
-        wght(iprop(1, 1)) = Material::GasWght[static_cast<int>(Material::GasType::Air)];
-        gama(iprop(1, 1)) = Material::GasSpecificHeatRatio[static_cast<int>(Material::GasType::Air)];
-        for (ICoeff = 1; ICoeff <= 3; ++ICoeff) {
-            gcon(ICoeff, iprop(1, 1)) = Material::GasCoeffsCon[ICoeff - 1][static_cast<int>(Material::GasType::Air)];
-            gvis(ICoeff, iprop(1, 1)) = Material::GasCoeffsVis[ICoeff - 1][static_cast<int>(Material::GasType::Air)];
-            gcp(ICoeff, iprop(1, 1)) = Material::GasCoeffsCp[ICoeff - 1][static_cast<int>(Material::GasType::Air)];
-        }
+        auto const &gasAir = Material::gases[(int)Material::GasType::Air];
+        wght(iprop(1, 1)) = gasAir.wght;
```

Comment:
I thought about going into `WindowComplexManager` and changing it to use the new `Material::Gas` structures directly, but then I woke up.